AdevintaSwiftUIPager.podspec

```diff
@@ -1,7 +1,7 @@
 Pod::Spec.new do |s|
 
   s.name         = "AdevintaSwiftUIPager"
-  s.version      = '2.0.0'
+  s.version      = '3.0.0'
```

Comment:
We can't use the same version as the original repo? In the future is gonna be difficult to understand that they are the same, won't it?

Reply:
yes, sure. My concerns where for us to be less restrictive than we should in the Podfile and updating it even with breaking changes but as you in Motor are the only ones using it for now... 😂